bokeh/tests/test_document.py

```diff
@@ -543,6 +598,33 @@ def cb(): pass
         assert isinstance(events[0], SessionCallbackAdded)
         assert isinstance(events[1], SessionCallbackRemoved)
 
+    def test_add_remove_next_tick_callback_with_args(self):
+        d = document.Document()
+
+        events = []
+        def listener(event):
+            events.append(event)
+        d.on_change(listener)
+
+        assert len(d.session_callbacks) == 0
+        assert not events
+
+        def cb(a,b,c,d): pass
+
+        callback = d.add_next_tick_callback(cb, 10, 20, c=30, d=40)
+        assert len(d.session_callbacks) == len(events) == 1
+        assert isinstance(events[0], SessionCallbackAdded)
+        assert callback == d.session_callbacks[0] == events[0].callback
+        assert callback._args == (10, 20)
+        assert callback._kwargs == dict(c=30, d=40)
+
+        callback = d.remove_next_tick_callback(cb)
+        assert len(d.session_callbacks) == 0
+        assert len(events) == 2
+        assert isinstance(events[0], SessionCallbackAdded)
+        assert isinstance(events[1], SessionCallbackRemoved)
+
+
```

Comment:
These tests just ensure that a proper "callback obj" is created that reflects any args/kwargs passed in. But it's only about half the journey.